### PR TITLE
Create capitalize.md

### PR DESCRIPTION
**Pull request by logankilpatrick**
Added capitalize example that shows the ternary operator.

### CODE REVIEW

File: snippets/capitalize.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: capitalize
+tags: string,intermediate
+---
+
+Capitalize the first letter of the string. 
+Set lower_rest to `true` if you want all but the first letter of the string converted to lowercase.
```
Comment (Chalarangelo) [inline snippets/capitalize.md]: Add an empty line before this.

File: snippets/capitalize.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: capitalize
+tags: string,intermediate
+---
+
+Capitalize the first letter of the string. 
+Set lower_rest to `true` if you want all but the first letter of the string converted to lowercase. 
+Omit the `lower_rest` parameter or set it to `false` to keep the rest of the string in its origional case.
```
Comment (Chalarangelo) [inline snippets/capitalize.md]: Please explain **how** the code works (`uppercasefirst()`, `lowercase` etc.).

File: snippets/capitalize.md (hunk 1 above)
Comment (logankilpatrick) [reply snippets/capitalize.md]: Before line 7?

Comment (Chalarangelo) [reply snippets/capitalize.md]: Yep, the parsing mechanism requires an empty line between the description and the explanation block.

Comment (logankilpatrick) [reply snippets/capitalize.md]: Got it. Does it look good now?

**Pull request merged**